### ISSUE TITLE
Time of Day column

### ISSUE THREAD

**Issue by awgymer** [mrcaseb/nflfastR#40]
With the release of the weather data by @DataWithBliss and @StatsbyLopez I was looking for a way to join data with real times to the play-by-play data which only has game time and kickoff time available, there is also `drive_real_start_time` but that seems to be entirely `NA`?. Maybe I'm missing something obvious but I couldn't see a reliable way to do this. 

Anyway looking at the raw JSON data it seems that there is a `'timeOfDay'` variable that appears to be real time in UTC on a play-by-play basis and appears to be available consistently for all games 2011 onwards.

How possible is it to get this value added into the dataset rather than scraping it out additionally manually?

**Comment by guga31bb** [comment]
Yep we will look at this, thanks for reminding us that it's there!

**Comment by awgymer** [comment]
Cool, I just started looking at it properly and I've realised that calculating the actual datetime is a clusterf%#@ because of timezones. For 1pm games it's relatively straightforward. For SNF and MNF the UTC times fall on the next calendar day. And I think worst of all there is the possibility of some games crossing over the calendar date during the game, making datetime calculations a pain. 

This probably isn't in the remit of nflfastR but it does make me wonder about the usefulness of the column after all.
